src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  transition: 0.3s;
+  width: 200px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  padding: 16px;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__button {
+    display: block;
+    padding: 11px 0;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+
+    border: 1px solid $blue-color;
+    background-color: $blue-color;
+    color: white;
+    text-decoration: none;
+    text-align: center;
+    text-transform: uppercase;
+    transition: 0.3s;
+  }
+
+  &__button:hover {
```

Comment:
Could be nested into `__button` [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=mixins%2C%20functions%2C%20etc.-,Use%20nesting)

Reply:
Eagle eyes! Great